misk-core/src/build.gradle

```diff
@@ -0,0 +1,27 @@
+buildscript {
+  dependencies {
+    classpath dep.kotlinNoArgPlugin
+  }
+}
+
+dependencies {
+  compile dep.guice
```

Comment:
meta: we should change these to `api`

Reply:
```
    When you switch to the new Android Gradle plugin 3.X.X, you should replace all your compile with the implementation keyword (1*). Then try to compile and test your app. If everything it's ok leave the code as is, if you have problems you probably have something wrong with your dependencies or you used something that now is private and not more accessible. Suggestion by Android Gradle plugin engineer Jerome Dochez (1)*)

    If you are a library mantainer you should use api for every dependency which is needed for the public API of your library, while use implementation for test dependencies or dependencies which must not be used by the final users.
```

- https://stackoverflow.com/questions/44413952/gradle-implementation-vs-api-configuration

Reply:
All of misk is doing this, and I think we should get to that later